lib/settings.py

```diff
@@ -166,14 +171,19 @@ def cmdline(command):
     I intend to have the issue resolved by Version 1.5.0.
     """
 
-    os.system(command)
-    '''process = subprocess.call(
-        args=" ".join(command),
-        stdout=subprocess.PIPE,
-        stderr=subprocess.STDOUT,
-        shell=True
-    )
-    return process'''
+    #os.system(command)
+    lib.output.info("Executing command {}".format(command))
```

Comment:
This command needs to be wrapped in single quotes and stripped example:

`executing command 'sudo msfconsole <SOMETHING> <OPTS>'`